src/game/ctrl.rs

```diff
@@ -122,27 +121,34 @@ impl GameController {
                     }
                 }
                 true
-            },
+            }
             Gesture::NoGesture => false,
-            _ => { panic!("Unimplemented gesture type."); }
+            _ => {
+                panic!("Unimplemented gesture type.");
+            }
```

Comment:
Could you please fix this up to just remove the braces and put the panic on the same line after `=>`? I imagine that, had there been no braces, that's what rustfmt would have done anyway.